src/app/pages/terms-of-service/terms-of-service.component.html

```diff
@@ -1,11 +1,18 @@
 <div class='container'>
   <h1 class='text-center mt-3 mb-3'>Terms of Service</h1>
   <div class='alert alert-primary' role='alert'>
-    Last updated: June 27, 2020
+    Last updated: July 15, 2020
```

Comment:
```suggestion
    Last updated: July 13, 2020
```

Reply:
It's not being published until the 15th - that's why the date is in the future.